### ISSUE TITLE
Get All Contents in Selective Deletion

### ISSUE THREAD

**Issue by NisiDominus** [seahindeniz/BrainlyTools_Extension#269]
A button can be added to list all content in selective content deletion.

* Button use can be constricted.

![image](https://user-images.githubusercontent.com/47514611/95336346-41275200-08b9-11eb-85a1-f840418dc458.png)

**Comment by seahindeniz** [comment]
Hey @NisiDominus 

Do you mean you need a way to select all contents across all pages based on the content type which you are on?

**Comment by NisiDominus** [comment]
@seahindeniz Yes, only 25 items are displayed on this page. I want the contents of all pages to be listed here.
